lib/twine-util/promises.ts

```diff
@@ -33,3 +33,7 @@ export const find = async <T>(fn: (a: T) => Promise<boolean>, xs: T[]): Promise<
 export const some = async <T>(ps: Promise<T>[]) => {
   return Promise.all(ps.map((p) => p.catch((error) => error)));
 };
+
+export const silent = <T>(p: Promise<T>): void => {
```

Comment:
Peer package installing is still broken. I could only get this working by traversing into this repo, building locally, then trashing & reinstalling `api`'s dependencies

Reply:
I added the postinstall script back (not sure when that was removed), but might be worth resurrecting #261